.github/workflows/ci.yaml

```diff
@@ -18,6 +18,7 @@ env:
   ARBITRUM_ONE_RPC_URL: ${{ secrets.ARBITRUM_ONE_RPC_URL }}
   SEPOLIA_RPC_URL: ${{ secrets.SEPOLIA_RPC_URL }}
   GOERLI_RPC_URL: ${{ secrets.GOERLI_RPC_URL }}
+  GNOSIS_CHAIN_RPC_URL: ${{ secrets.GNOSIS_CHAIN_RPC_URL }}
```

Comment:
_(arbitrary comment location to start a thread)_

One last thing I thought of is we should update the readme to document the deploys/addresses of periphery contracts

Reply:
Seems we don't even have the core Umbra or registry address in the contract READMEs, maybe worth adding those too

Reply:
done in bcbb335bfd2360ac16799fe0c1e0d47cdd6c43ca